environments/ui-draft.yaml

```diff
@@ -0,0 +1,34 @@
+version: "3.7"
+x-env-file-mount:
+  &env-file-mount
+  type: bind
+  source: .env
+
+networks:
+  source-verify:
+
+x-project-base:
+  &project-base
+  env_file:
+  - .env
+  restart: always
+  networks:
+  - source-verify    
+
+services:
+  ui-draft:
+    <<: *project-base
+    build:
```

Comment:
This should be replaced by `image` because you are building and pushing that image before and this yaml is only used for deployment. 
This was good for local testing :)

Reply:
ah